java/arcs/core/policy/PolicyConstraints.kt

```diff
@@ -43,8 +52,63 @@ fun translatePolicy(policy: Policy, recipe: Recipe): PolicyConstraints {
             }
     }
 
-    // TODO(b/157605232): Add store claims.
-    return PolicyConstraints(policy, recipe, egressChecks)
+    // Add claim statements for stores.
+    val targetBySchemaName = policy.targets.associateBy { it.schemaName }
+    val storeClaims = recipe.handles.values.filter { storeMap.containsKey(it.id) }
+        .associateWith { handle ->
+            val schemaName = storeMap[handle.id] ?: return@associateWith emptyList<Claim>()
```

Comment:
If two stores have the same schema, do we still apply the claims if it is the `storeMap`?

Essentially, it looks like `storeMap` is our `@protected` annotated stores?

Reply:
Yep, that's right. I thought that was the plan?

Reply:
Yep, the plan was to keep a map of some kind. When we were trying to map schemas to store ids automatically, we wanted to make sure that there is only one store for a particular schema. Your approach is more flexible, which is fine. Just wanted to check.